sys/compat/linux/linux_misc.c

```diff
@@ -1819,6 +1819,14 @@ linux_prctl(struct thread *td, struct linux_prctl_args *args)
 #endif
 		error = EINVAL;
 		break;
+	case LINUX_PR_SET_CHILD_SUBREAPER:
+		if (args->arg2 == 0){
```

Comment:
We’re just going to ignore style(9) here?

Reply:
> We’re just going to ignore style(9) here?

Please state the problem rather than assume intent.

Stlye(9) is a huge pain in the ass to get right all the time. My whiny script about didn't catch this lack of space and I didn't see it until you pointed it out just now.

Reply:
My point was that your handy script *did* fail here:

<img width="685" alt="image" src="https://github.com/freebsd/freebsd-src/assets/816232/f7f7b0fb-d172-4235-9d5d-17a11fb120d1">
<img width="851" alt="image" src="https://github.com/freebsd/freebsd-src/assets/816232/934ac8a9-9e28-4e77-a3ae-ea2b24f3a1cd">

Reply:
But sorry for being grumpy, I was just annoyed that I saw the style issue in the commit in my inbox and that, when I went to check the PR, it had been clearly flagged by the automation

Reply:
Somehow it wasn't flagged by the script I was using... Oh... I see what went wrong with my scripting...